bundles/org.eclipse.equinox.common/src/org/eclipse/core/runtime/Path.java

```diff
@@ -1116,6 +1116,14 @@ public IPath setDevice(String value) {
 	public File toFile() {
 		return new File(toOSString());
 	}
+	
+	/* (Intentionally not included in javadoc)
+	 * @see IPath#toPath()
+	 */
+	@Override
+	public java.nio.file.Path toPath() {
+		return java.nio.file.Path.of(toOSString());
```

Comment:
Why not put this implementation directly in IPath.toPath() where org.eclipse.core.runtime.IPath.toOSString() is available?

Reply:
Oh. I was blindly following the IPath -> Path pattern. Do we still need the implementation in Path then?

Reply:
A single implementation via the default method should be sufficient.  😀

Reply:
As IPath is marked as noimplement and noextend I would just put it into the (single) implementation.

Reply:
In that case I think either way is fine and there is no great difference.